app/src/main/java/de/danoeh/antennapod/ui/screen/feed/preferences/FeedSettingsFragment.java

```diff
@@ -462,25 +462,21 @@ private void setupKeepUpdatedPreference() {
             });
         }
 
-        private void setupAutoDownloadGlobalPreference() {
-            if (!UserPreferences.isEnableAutodownload()) {
-                SwitchPreferenceCompat autodl = findPreference("autoDownload");
-                autodl.setChecked(false);
-                autodl.setEnabled(false);
-                autodl.setSummary(R.string.auto_download_disabled_globally);
-                findPreference(PREF_EPISODE_FILTER).setEnabled(false);
-            }
-        }
-
         private void setupAutoDownloadPreference() {
-            SwitchPreferenceCompat pref = findPreference("autoDownload");
+            SwitchPreferenceCompat pref = findPreference(PREF_AUTODOWNLOAD);
```

Comment:
The preference should be a 3-state choice: Yes, No, Global. Similar to the auto-delete setting on the same screen.